yarn.lock

```diff
@@ -3329,7 +3329,7 @@
   integrity sha512-S9K+Q39BGOghyLHmR0Wdcmu1i1noSUk8HcvMj+3IaohZw02WFd99aPTQDHJeseXrXZP3CNovaSlePI0R11NcFg==
 
 "@cds/core@./dist/core":
-  version "5.0.0-rc.0"
+  version "5.0.0-rc.1"
```

Comment:
Should this and the other locked versions below be `5.0.1`?

Reply:
Probably need just to merge with `next` .. it's an old PR - before the `v5` release.